docs/units-and-global-variables.rst

```diff
@@ -120,11 +120,11 @@ Error Handling
 --------------
 
 ``assert(bool condition)``:
-    throws if the condition is not met - to be used for internal errors.
+    invalidates the transaction if the condition is not met - to be used for internal errors.
```

Comment:
Why did you not use `reverts` here? The effects are a little different, yes, but because of the 63/64-rule, it does not use up all gas and such errors might even get caught further up the call chain, so it does not necessarily revert the full transaction.

Reply:
yes that was the reason. did not know a good alternative word so i asked on gitter solidity chat and that was a suggestion i got.

Reply:
Since it calls INVALID it kind of made sense in that way too.

Reply:
Alternatively: aborts or exceptionally aborts

Though I am fine with the way it is.

Reply:
Sorry for being indecisive, but after some consideration, I think overall `exceptionally aborts` may be a better choice because it is clear an abort happens (and not an "invalidation") and searching for exception or exceptionally in the Yellowpaper will lead to the correct section (FWIW).

Reply:
Note that this is just the short summary, we should check that we use the same wording as in the more extensive section.

Reply:
Created an issue for it https://github.com/ethereum/solidity/issues/4099